omero/sysadmins/unix/install-web/walkthrough/omeroweb-install-centos7-ice3.6.rst

```diff
@@ -97,6 +97,14 @@ Install the OMERO.web requirements. Select one of the commands corresponding to
     
     
 
+Installing OMERO.web apps
+-------------------------
+
+**The following steps are run as the omero system user.**
```

Comment:
I wonder if these headings ought to put "OMERO" in all-caps.

Reply:
you mean the omero system user.
This was discussed few times and decision was to indicate for each which will do the tasks either omero or root

Reply:
Oh, might deserve backticks or something then if it's supposed to be a literal username, that wasn't at all apparent to me. That issue's outwith this PR though.

Reply:
(Taking it as "omero system user" means "operating system user named 'omero'" rather than "user of the OMERO system".)

Reply:
"operating system user named 'omero'"  that's correct